### ISSUE TITLE
The builds for Raspbian_11 on OBS are broken

### ISSUE THREAD

**Issue by jloehel** [zeek/zeek#4087]
Can someone please check the builds for [Raspbian_11](https://build.opensuse.org/projects/security:zeek/packages/zeek/repositories/Raspbian_11/binaries). The repo is broken.

**Comment by timwoj** [comment]
Can you give some more information? It looks like the build succeeded. I'm able to look at the repository over http, but I admittedly don't have a rPi here to test the repo with.

**Comment by jloehel** [comment]
> Can you give some more information? It looks like the build succeeded. I'm able to look at the repository over http, but I admittedly don't have a rPi here to test the repo with.

I guess the publishing failed. The build itself looks good. I had issues with gpg keys and I get a 404s:
~~~
E: Failed to fetch https://download.opensuse.org/repositories/security:/zeek/Raspbian_11/armhf/zeek_7.0.1-0_armhf.deb  404  Not Found [IP: 195.135.223.226 443]
E: Failed to fetch https://download.opensuse.org/repositories/security:/zeek/Raspbian_11/armhf/zeekctl_7.0.1-0_armhf.deb  404  Not Found [IP: 195.135.223.226 443]
E: Failed to fetch https://download.opensuse.org/repositories/security:/zeek/Raspbian_11/all/zeek-zkg_7.0.1-0_all.deb  404  Not Found [IP: 195.135.223.226 443]
E: Failed to fetch https://download.opensuse.org/repositories/security:/zeek/Raspbian_11/all/zeek-btest_7.0.1-0_all.deb  404  Not Found [IP: 195.135.223.226 443]
E: Failed to fetch https://download.opensuse.org/repositories/security:/zeek/Raspbian_11/all/zeek-btest-data_7.0.1-0_all.deb  404  Not Found [IP: 195.135.223.226 443]
E: Failed to fetch https://download.opensuse.org/repositories/security:/zeek/Raspbian_11/armhf/zeek-spicy-dev_7.0.1-0_armhf.deb  404  Not Found [IP: 195.135.223.226 443]
E: Failed to fetch https://download.opensuse.org/repositories/security:/zeek/Raspbian_11/all/zeek-client_7.0.1-0_all.deb  404  Not Found [IP: 195.135.223.226 443]
E: Failed to fetch https://download.opensuse.org/repositories/security:/zeek/Raspbian_11/armhf/zeek-core_7.0.1-0_armhf.deb  404  Not Found [IP: 195.135.223.226 443]
E: Failed to fetch https://download.opensuse.org/repositories/security:/zeek/Raspbian_11/armhf/zeek-core-dev_7.0.1-0_armhf.deb  404  Not Found [IP: 195.135.223.226 443]
E: Failed to fetch https://download.opensuse.org/repositories/security:/zeek/Raspbian_11/armhf/libbroker-dev_7.0.1-0_armhf.deb  404  Not Found [IP: 195.135.223.226 443]
~~~

**Comment by awelzel** [comment]
Could you run `apt-get update` (if it's in a container, make sure to bust the build cache). Your output indicates downloading 7.0.1 - these are not available anymore, only version 7.0.4 is in the repos.
